client/static/js/lib/handlers.js

```diff
@@ -0,0 +1,45 @@
+const main = document.querySelector('main');
+const { getPostData, createNewPost } = require('./fetch');
+const { createFormHTML, createPost, renderPostNotFound } = require('./helpers');
+
+async function handlePostFormSubmit(e) {
+	try {
+		e.preventDefault();
+		const body = Object.fromEntries(new FormData(e.target));
+		const postData = await createNewPost(body);
+		if (!postData.id) {
+			throw new Error('New post has no id.');
+		}
+		window.location.hash = postData.id;
+	} catch (err) {
+		console.error(err);
+	}
+}
+
+async function updateMain(hash) {
+	main.textContent = '';
+	let child;
+	if (hash) {
+		const postData = await getPostData(hash.slice(1));
+		if (!postData.id) {
+			renderPostNotFound();
+			return;
+		}
+		child = createPost(postData);
+	} else {
+		child = createFormHTML(handlePostFormSubmit);
+	}
+	main.append(child);
+}
+
+function updateConent() {
```

Comment:
Hey Max, I'm assuming this ones you mate after our project last week. Get the spelling in there!